hooks/checkout

```diff
@@ -371,10 +388,8 @@ setup_git_lfs() {
     # Also setting both https and http URLs, so that if git-proxy requires http URL as remote it should still work.
     if [[ "${CANVA_REPO}" =~ ^https?://.* ]]; then
       git config "lfs.https://github.com/${CANVA_REPO#*@github.com/}/info/lfs.access" basic
-      git config "lfs.http://github.com/${CANVA_REPO#*@github.com/}/info/lfs.access" basic
     else
       git config "lfs.https://github.com/${CANVA_REPO#*@github.com:}/info/lfs.access" basic
-      git config "lfs.http://github.com/${CANVA_REPO#*@github.com:}/info/lfs.access" basic
     fi
```

Comment:
do we still need this logic here? considering this is set in the `checkout` method.

Reply:
Yes, we still need them. The checkout method only configures the `lfsurl` and `lfspushurl` endpoints. But we still need to set the auth method to basic here to avoid API rate limiting. But the good news is, we don't need the token here.

To verify the two work together, you can run `git lfs env` and it should show
```
Endpoint=https://x-access-token:ghs_12345@github.com/Canva/canva.git/info/lfs (auth=basic)
```